server/jetstream_api.go

```diff
@@ -363,6 +363,7 @@ type JSApiStreamListResponse struct {
 	ApiResponse
 	ApiPaged
 	Streams []*StreamInfo `json:"streams"`
+	Missing []string      `json:"missing_stream_names,omitempty"`
```

Comment:
This is a "stream list response" object, so I think "missing,omitempty" should be enough (no need for `_stream_names`)